niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -2360,7 +2444,11 @@ Transacts.cancelAuthenticationFromService, getActivityToken(),
                         mTransacts.invokeTransact(Transacts.STATUS_BAR_SERVICE,
                                 Transacts.STATUS_BAR_DESCRIPTOR,
                                 Transacts.hideBiometricDialog);
-                    } else {
+                    } else if (mDeviceApiLevel == Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Does this only apply to Android T, or should this check be >= T?

Reply:
Thanks. Changed it to >=T

Reply:
Change looks good; thanks!